sdk/communication/azure-communication-phonenumbers/pom.xml

```diff
@@ -49,8 +49,8 @@
   <properties>
     <src.dir>src/main</src.dir>
     <test.dir>src/test</test.dir>
-    <jacoco.min.linecoverage>0.68</jacoco.min.linecoverage>
-    <jacoco.min.branchcoverage>0.66</jacoco.min.branchcoverage>
+    <jacoco.min.linecoverage>0.70</jacoco.min.linecoverage>
```

Comment:
We skip some tests related to Purchase and Release in our live test pipeline but not locally. Updating the coverage here may cause failures in the live test pipeline as coverage may not be met